notebooks/ullyses-env-test.ipynb

```diff
@@ -0,0 +1,241 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "19a052a8-d7d7-46bc-8280-802b513946bb",
+   "metadata": {},
+   "source": [
+    "<a id=\"top\"></a>\n",
+    "# Environment Set up Test for ULLYSES notebooks\n",
+    "***\n",
+    "\n",
+    "This notebook can be used to ensure that everything in your enviornment has been installed correctly. Instructions about how to install conda and the stenv environment can be found at: https://stenv.readthedocs.io/en/latest/getting_started.html. Once conda is installed, follow the documentation outlined in the [README](https://github.com/spacetelescope/ullyses?tab=readme-ov-file#installation) of this repository. If everything is working properly, this notebook should run to completion in each cell without any errors and the final cell should have a ullyses version >= 4.1.0 and a ullyses_utils version of >= 3.1.0."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cc5107a5-179e-45d9-a50b-514f8ebd1f68",
+   "metadata": {},
+   "source": [
+    "First, ensure you are in the conda enviornment that you have created. If you are not in the correct environment, use the command `conda activate <env_name>` to switch into the correct enviornment. Also ensure that your python version is 3.9 or later."
```

Comment:
This didn't actually work for me in jupyter. I think you have to do it on the terminal and restart the kernel.

Reply:
I think it's still useful to have this line to check the environment, but I can make a quick change to say that this needs to be done outside of jupyter notebook